tests/ui/type-alias-impl-trait/nested-tait-hrtb.rs

```diff
@@ -8,7 +8,7 @@ fn without_lt() -> impl for<'a> Trait<'a, Assoc = WithoutLt> {}
 //~^ ERROR captures lifetime that does not appear in bounds
 
 type WithLt<'a> = impl Sized + 'a;
-//~^ ERROR concrete type differs from previous defining opaque type use
+
```

Comment:
Why does this disappear? It's a TAIT, not RPIT?

Reply:
it's used in an RPIT right below